src/ray/stats/metric.h

```diff
@@ -142,6 +142,16 @@ class Sum : public Metric {
 
 };  // class Sum
 
+/// Raw metric view point for exporter.
+struct MetricPoint {
+  std::string metric_name;
+  int64_t timestamp;
+  double value;
+  std::unordered_map<std::string, std::string> tags;
```

Comment:
Any particular reason why we don't use `absl::flat_hash_map` instead?

Reply:
> Any particular reason why we don't use `absl::flat_hash_map` instead?


For compatibility considerations, avoid redundant data conversion bettween exporter and client. Absl may not be introduced in other tool libraries.